### DISCUSSION TITLE
Error post install trying to get directory information

### DISCUSSION THREAD

**Discussion by barroei1981** [Blazam-App/BLAZAM#463] [Q&A] [answered]
[Error] Current User State was not provided to this directory entry ApplicationException { TargetSite: BLAZAM.Session.Interfaces.IApplicationUserState get_CurrentUser(), Message: "Current User State was not provided to this directory entry", Data: [], InnerException: null, HelpLink: null, Source: "BLAZAMActiveDirectory", HResult: -2146232832, StackTrace: "   at BLAZAM.ActiveDirectory.ActiveDirectoryContext.get_CurrentUser() in D:\a\BLAZAM\BLAZAM\BLAZAMActiveDirectory\ActiveDirectoryContext.cs:line 32
   at BLAZAM.ActiveDirectory.Adapters.DirectoryEntryAdapter.get_CurrentUser() in D:\a\BLAZAM\BLAZAM\BLAZAMActiveDirectory\Adapters\DirectoryEntryAdapter.cs:line 101

i am logged in with the Admin user, and i cant set any additional users ince i get find anything on the AD Search

**Comment by jacobsen9026** [comment]
Welcome @barroei1981, and I apologize for the error.

I think this warrants an issue, please continue this discussion in #464 

I will conduct a preliminary investigation and report back there within 24 hours.

**Comment by jacobsen9026** [comment, marked as answer]
For archival purposes, the solution was to install the WebSockets component of IIS
